sdkutil/address.go

```diff
@@ -0,0 +1,10 @@
+package sdkutil
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// GetAccAddressFromBech32 creates an AccAddress from a Bech32 string.
+// It internally calls `sdk.AccAddressFromBech32` and ignores the error.
+func GetAccAddressFromBech32(address string) sdk.AccAddress {
+	addr, _ := sdk.AccAddressFromBech32(address)
```

Comment:
should we set here panic if address is invalid?

Reply:
I had done that initially, but, that causes `TestCreateBidNonExistingOrder` to fail here: https://github.com/ovrclk/akash/blob/cosmos-upgrade/x/market/handler/handler_test.go#L134 because it passes an empty string for order owner and provider address.

I guess, we can set those strings to a valid address that doesn't exist in the store to make the test pass.

Reply:
Done